working/wildcards/feature-specification.md

```diff
@@ -244,6 +255,39 @@ class C {
 }
 ```
 
+### Super parameters
+
+An occurrence of `super._` as a declaration of a formal parameter in a
+constructor is a compile-time error.
+
+*It is not an error everywhere: In a method body it could be an invocation
+of an inherited getter named `_`.*
+
+*The desugared meaning of a super parameter includes a reference to the
+parameter in the initializer list of the enclosing constructor declaration,
+but such references are not possible when the parameter name is a
+wildcard.*
+
+### Extension types
+
+An extension type declaration has a `<representationDeclaration>`
+which is similar to a formal parameter list of a function declaration.
+
+*It always declares exactly one mandatory positional parameter, and the
+meaning of this declaration is that it introduces a formal parameter of a
+constructor of the enclosing extension type as well as a final instance
+variable declaration, also known as the representation variable of the
+extension type.*
+
+This parameter can have the declared name `_`. This means that the
+representation variable is named `_`, and no formal parameter name is
+introduced into any scopes.
+
+*Currently that parameter is not in scope for any code anyway, but future
```

Comment:
If I'm understanding correctly, if we have primary constructors, the formal parameter `_` will be inaccessible anyways. Could we write that more explicitly?

Reply:
I gave an example where it does matter, but it involves both primary constructors and the hypothetical extension which allows superconstructor invocations to be specified in the header. So it's a pretty remote issue, and it might work better to just omit it.

Anyway, it's there now. Take a look! ;-)

Reply:
OK, I deleted those 20 lines.